### PR TITLE
GOVUKAPP-590 Global ON/OFF switch

### PR DESCRIPTION
**Pull request by TimGJD**
# Title
Global ON / OFF switch

# Description
Add app unavailable screen and the logic to show it

## JIRA ticket(s)
  - [GOVAPP-590](https://govukverify.atlassian.net/browse/GOVUKAPP-590)

## Figma
  - [Figma board](https://www.figma.com/design/5u5cvVCgKY3Rng2ADNV30K/2024-07---GOV.UK-app---beta-designs?node-id=363-6730&node-type=section&t=ZK9eNXBoSekBxAnK-0)

### After

| Light Mode | Dark Mode |
|---|---|
| image | image |

### CODE REVIEW

File: feature/availability/src/main/kotlin/uk/govuk/app/availability/ui/UnavailableScreen.kt
Comment (gclssvglx) [reply feature/availability/src/main/kotlin/uk/govuk/app/availability/ui/UnavailableScreen.kt:68]: Ah, `govenor` is a known issue with talkback. 

If it's visible to the user, then I believe we've accepted that this might happen and have chosen to proceed with it rather than try to fix it everywhere. 

If `GOV.UK` is not visible - just read by talkback - it's ok to replace `GOV.UK` with `GOV UK` (space, not dot).

**Review by gclssvglx — DISMISSED** [review]
This is looking good to me 👍 

However, I'd wait for a review from @davidc-gds before merging

**Comment by davidc-gds** [comment]
Looks like sonar has flagged some unused imports that need removing. I wouldn't worry about the other issues.

File: app/src/main/kotlin/uk/govuk/app/ui/GovUkApp.kt
Comment (davidc-gds) [inline app/src/main/kotlin/uk/govuk/app/ui/GovUkApp.kt:191]: I've introduced an app launch navigation class to tidy this up. Might be worth waiting until my changes are merged and then integrating this with it.

File: app/build.gradle.kts
Comment (davidc-gds) [inline app/build.gradle.kts:94]: Not sure this needs to be a feature module, feels like more of a function of the app module itself?

Comment (TimGJD) [reply app/build.gradle.kts:94]: Yes, I see what you mean. I've got 3 related screens going in (this unavailable one, forced update and recommend update) which is why I thought I'd pop them in a module together. I will move to main app module.

File: app/src/main/kotlin/uk/govuk/app/navigation/AppLaunchNavigation.kt
Comment (TimGJD) [inline app/src/main/kotlin/uk/govuk/app/navigation/AppLaunchNavigation.kt:36]: As App Unavailable has no further in-app navigation after it I have wondered if perhaps we shouldn't push the other graph routes in this scenario or maybe it doesn't matter and is simpler to leave as-is?

File: app/src/main/kotlin/uk/govuk/app/ui/GovUkApp.kt
Comment (TimGJD) [reply app/src/main/kotlin/uk/govuk/app/ui/GovUkApp.kt]: Done

File: app/build.gradle.kts
Comment (TimGJD) [reply app/build.gradle.kts]: Done

File: app/src/main/kotlin/uk/govuk/app/navigation/AppLaunchNavigation.kt
Comment (davidc-gds) [reply app/src/main/kotlin/uk/govuk/app/navigation/AppLaunchNavigation.kt:36]: I've had to re-jig this a bit anyway due to the bug we found during post amigo so I'll sort it out post merge

**Review by davidc-gds — APPROVED** [review]

**Pull request merged**
